docs/conf.py

```diff
@@ -68,6 +68,12 @@
 html_extra_path = ['./.nojekyll', './.gitattributes']
 html_favicon = 'favicon.png'
 
+html_context = {
```

Comment:
The latest versions of Sphinx & rtd-docs don't support adding css files like this. See https://github.com/readthedocs/sphinx_rtd_theme/issues/1415

Need to use `html_css_files`

Reply:
Thanks - updated to use `html_css_files